modules/playing-cards/module.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace Elementor\Modules\PlayingCards;
+
+use Elementor\Core\Base\Module AS BaseModule;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class Module extends BaseModule {
+	public function get_name() {
+		return 'playing-cards';
+	}
+
+	protected function get_widgets() {
+		return [
+			'Playing_Cards',
+		];
+	}
+	public function __construct() {
+		parent::__construct();
+		add_action( 'elementor/frontend/after_register_scripts', [ $this, 'register_scripts' ] );
+		add_action( 'elementor/frontend/after_register_styles', [ $this, 'register_styles' ] );
+	}
+
+	public function register_styles() {
+		$direction_suffix = is_rtl() ? '-rtl' : '';
+		$has_custom_breakpoints = Plugin::$instance->breakpoints->has_custom_breakpoints();
+
+		$widget_style_name = 'widget-playing-cards';
+
+		wp_register_style(
+			$widget_style_name,
+			$this->get_frontend_file_url( "{$widget_style_name}{$direction_suffix}.min.css", $has_custom_breakpoints ),
+			[ 'elementor-frontend', 'elementor-icons' ],
```

Comment:
Do you have deps on the icons?

```suggestion
			[ 'elementor-frontend' ],
```

Reply:
good to know, I thought that if a widget (PHP class) uses icons, they need to be loaded first. but yes, you're right, these are styles for the frontend, and they are not necessary